### PR TITLE
Treat DMI information as optional

### PR DESCRIPTION
**Pull request by dirkmueller**
Some (new) hardware is not supported by dmidecode, so the
dmi tree is missing in ohai. Just ignore it, since the
whole point of that is logging the information.

### CODE REVIEW

**Comment by tboerger** [comment]
IMHO this logging line is anyway useless but i'm fine with this change.

File: crowbar_framework/app/models/node_object.rb [hunk 1]
```diff
@@ -1335,7 +1335,8 @@ def unique_device_for(device)
 
       # VirtualBox does not provide stable disk ids, so we cannot rely on them
       # in that case.
-      unless @node[:dmi][:system][:product_name] =~ /VirtualBox/i
+      dmisystem=@node[:dmi][:system][:product_name] rescue "unknown"
```
Comment (vuntz) [inline crowbar_framework/app/models/node_object.rb]: Style nitpick: missing spaces around `=`

**Comment by vuntz** [comment]
+1

**Comment by tboerger** [comment]
LGTM

**Pull request merged**